commerce_coordinator/apps/commercetools/sub_messages/tasks.py

```diff
@@ -263,18 +280,19 @@ def _prepare_segment_event_properties(in_order, return_line_item_return_id):
 
         if 'refund_response' in result and result['refund_response']:
             if result['refund_response'] == 'charge_already_refunded':
-                logger.debug(f'[CT-{tag}] payment intent {payment_intent_id} already has refund transaction, '
-                             f'sending Zendesk email')
+                logger.info(f'[CT-{tag}] payment intent {payment_intent_id} already has refund transaction, '
```

Comment:
Please update the [test patch](https://github.com/edx/commerce-coordinator/blob/main/commerce_coordinator/apps/commercetools/tests/sub_messages/test_tasks.py#L322) to `info` instead of `debug`